### DISCUSSION TITLE
React-Native-Google-Wallet Pass Implementation

### DISCUSSION THREAD

**Discussion by Fkhuja** [react-native-community/discussions-and-proposals#675] [Ideas]
Hello everybody. I have been working on trying to implement adding a pass to google wallet for my android device. I am trying to implement this on react native. Has anyone worked on this feature before and if so did you use any external npm packages? If completed what documentation did you use? Please let me know.

**Comment by PurpleMonkey729** [comment]
@Fkhuja ,
I think for you to prefer to make an issue of your idea in this repository.

Best wishes!

**Comment by kelset** [comment]
👋 there - as stated in the README and the issue template, this repo is for long-form discussions. Yours seems more a question for [StackOverflow instead](https://stackoverflow.com/questions/tagged/react-native).
